Changelog for 1.0.6.3 (#2135)
btcpayserver/btcpayserver: Changelog.md

# Changelog

## 1.0.6.3:

### New features

> * Greenfield API: Can update invoice metadata @Kukks

Reviewer: ```suggestion
* Greenfield API: Can update invoice metadata @Kukks
* Greenfield API: Notifications API
```

* Greenfield API: Can specify the preferred language when creating an invoice @NicolasDorier
* Greenfield API: Can specify the URL to redirect customer to when he paid when creating an invoice @NicolasDorier
* Greenfield API: Expose the `checkoutLink` of the created invoice, where you need to redirect your customer to pay in BTC @NicolasDorier
* Add a `Mark all as seen` button to the notification dropdown @ubolator
* Support of Armenian language in checkout page. Thanks to Mash Mashok
* Add ability to add custom CSS to pull payments @ubolator
* Introduce some basic spam protection for hosts with open registration (see #2106) @Kukks

### Improvements

> * Hide pagination & page size when not necessary  (#2122) @dennisreimann

Reviewer 2: ```suggestion
* Hide pagination & page size when not necessary (#2122) @dennisreimann @dstrukt
```

* Document in `/docs` miscallenous features of BTCPay (such as optional arguments of the checkout page) @NicolasDorier

> * UI: Improve mobile store setup view (according to [this comment](https://github.com/btcpayserver/btcpayserver/pull/2117#issuecomment-741255465)) @dennisreimann

Reviewer 2: ```suggestion
* UI: Improve mobile store setup view @dennisreimann @dstrukt
```

* Improve U2F support, should leverage OS support and now work properly on mobile safary (See #2086) @dennisreimann

> * UI: Store setup font finetuning (see #2087) @dennisreimann

Reviewer 2: We can remove this one, it was just a follow-up to the previous store setup improvement (line 21 here)

```suggestion
```

* Improve how new label picked their color (See #2072) @ubolator
* Improve the design of transaction's label dropdown to fix display issue when there are too many (see #2078) @ubolator

### Bug fixes:

* Ensure campaign status is shown correctly in minimal crowdfund app (#2062) @ubolator
* Can remove automatic labels from invoices @NicolasDorier
* Fix Pay Button Link preview when app mode chosen (#2115) @Kukks
* If the user was not admin, the API Keys page was ignoring some of the checked permissions @NicolasDorier

> * Greenfield API: If invoice creation failed for some problem, BTCPay would give a HTTP 500 error without details helping the user. @NicolasDorier

Reviewer 3: ```suggestion
* Greenfield API: If invoice creation failed for some reason, BTCPay would give a HTTP 500 error without details helping the user. @NicolasDorier
```

> * Fix performance issue: Some invoice query were causing a full table scan on all invoices rather than using an index. @NicolasDorier

Reviewer 3: ```suggestion
* Fix performance issue: Some invoice queries were causing a full table scan on all invoices rather than using an index. @NicolasDorier
```

> * Fix: Importing a wallet from BlueWallet was crashing (#2098) @Kukks

Reviewer: ```suggestion
* Fix: Importing an incorrect wallet from BlueWallet was crashing (#2098) @Kukks
```

* Fix classic theme for payment requests (Fix #2084) @dennisreimann

> ### Miscalleneous:

Author: damn I have been wrong on this one all my life!

* Removing the bitpay invoice translator @NicolasDorier
* Improve the documentation of API Keys page @NicolasDorier
* Fix doc for create invoice request's metadata @NicolasDorier

> 

Reviewer: ```suggestion
Fix docs for pull payments @Kukks
```

# Changelog

## 1.0.6.3:

### New features

* Greenfield API: Can update invoice metadata @Kukks
* Greenfield API: Can specify the preferred language when creating an invoice @NicolasDorier
* Greenfield API: Can specify the URL to redirect customer to when he paid when creating an invoice @NicolasDorier
* Greenfield API: Expose the `checkoutLink` of the created invoice, where you need to redirect your customer to pay in BTC @NicolasDorier
* Add a `Mark all as seen` button to the notification dropdown @ubolator
* Support of Armenian language in checkout page. Thanks to Mash Mashok
* Add ability to add custom CSS to pull payments @ubolator
* Introduce some basic spam protection for hosts with open registration (see #2106) @Kukks

### Improvements

> * The QR Code is now showing `bitcoin:BECH32` which seems the most compatible scheme improving QR readability (See our study #2110 on 40 wallets) @rockstardev

Reviewer 4: Study still in progress... I haven't implemented it, waiting on @pavlenex approval. Unless you coded it @NicolasDorier then we can remove it from this change log.
